release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -63,6 +63,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-networking"]
 === Networking
 
+[id="ocp-4-10-hardware"]
+=== Hardware
+
+[id="ocp-4-10-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+In a bare metal cloud environment, applications must act on hardware changes and failures quickly with high reliability. OpenShift Container Platform now provides a hardware event proxy that enables such applications to respond to Redfish hardware events with low latency.
```

Comment:
Is "bare metal cloud" an oxymoron? Also, "bare metal" in the adjectival postion takes a hyphen.
 
```suggestion
{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events such as hardware changes and failures.
```